README.md

```diff
@@ -63,53 +70,56 @@ _We do not recommend using the default options for PostgreSQL. The above values
 - `DATABASE_HOST`: The IP address Docker is running on. You can find this by running `docker-machine ip` but it's usually `192.168.99.100` instead of `localhost`
 - `DATABASE_URL_DEV`: This will need to be adjusted as well, for example `DATABASE_URL_DEV=postgres://postgres:.@localhost:5432/postgres` would become `DATABASE_URL_DEV=postgres://postgres:.@192.168.99.100:5432/postgres`
 
-### PostgreSQL
+## 2.3. PostgreSQL
 ***You will need a PostgreSQL database running locally to run this application locally.*** You may setup PostgreSQL however you wish, however we recommend using Docker using the instructions found here: https://hub.docker.com/_/postgres
 
 If you are using the Docker method you may spin up your database layer by running this command:
 ```
-docker run -d -e POSTGRES_HOST_AUTH_METHOD=trust -p 5432:5432 postgres
+docker run -d -e POSTGRES_PASSWORD=<your chosen password> -p 5432:5432 postgres
 ```
 
-_Note that POSTGRES_HOST_AUTH_METHOD=trust is insecure and will allow anything to connect to that db! Only use for testing and/or local dev!_
-
 If you're running a database in another way then we trust you can sort it out on your own because you're awesome :sunglasses:
 
-### Sequelize _<optional>_
-You can run the application without doing anything and it will create the tables needed to operate automatically. It will not, however, create users. If you would like to seed your database with users you will need to follow a few steps.
-1. You may create your database tables without running the application by running `npm run db-create`.
-2. You can now seed your database by running `npm run db-seed`. 
+### 2.3.1. Sequelize
+To properly start the application the database needs to be built by Sequlize ahead of time. To do that run the following commands
+1. You must create your database tables without running the application by running `npm run db-create` first.
+2. _optional_ You can now seed your database if you wish by running `npm run db-seed`. 

```

Comment:
I'm getting an error on npm run db-create: 'Dialect needs to be explicitly supplied as of v4.0.0'

Is this due to Postgres not being defined where it was previously (.env?, Dockerfile?)

Reply:
You have to have your `DATABASE_URL` defined in the `.env` file. The URL contains the un/pw, host, db, etc.